packages/block-editor/src/components/block-settings-menu/block-settings-dropdown.js

```diff
@@ -176,6 +177,13 @@ export function BlockSettingsDropdown( {
 										{ __( 'Move to' ) }
 									</MenuItem>
 								) }
+								{ count === 1 && (
+									<BlockLockMenuItem
```

Comment:
I feel this control should be moved to the following group (with reusable block, group, etc) since it's an action that modifies the block state.

cc @critterverse @jasmussen

Reply:
I agree — updated in [the Figma](https://www.figma.com/file/CUZSVCxP0sDGlP8aInzGaO/User-level-Block-Locking?node-id=20%3A2763)

cc @Mamaduka

Reply:
Updated.